src/client/index.ts

```diff
@@ -61,5 +62,30 @@ const hasCookie = (key: string, options?: OptionsType): boolean => {
   return Object.prototype.hasOwnProperty.call(cookies, key);
 };
 
+const useWrappedCookieFn = <TCookieFn extends (...args: any) => any>(cookieFnCb: TCookieFn) => {
+  const [isMounted, setIsMounted] = useState(false);
+  useEffect(() => {
+    setIsMounted(true);
+  }, []);
+  return isMounted ? cookieFnCb : ((() => {}) as TCookieFn);
+};
+
+const useGetCookies = () => useWrappedCookieFn(getCookies);
+const useGetCookie = () => useWrappedCookieFn(getCookie);
+const useHasCookie = () => useWrappedCookieFn(hasCookie);
+const useSetCookie = () => useWrappedCookieFn(setCookie);
+const useDeleteCookie = () => useWrappedCookieFn(deleteCookie);
```

Comment:
Hi! 
I'm so sorry for the huge delay in the review!

How about only one hook for all functions? 

Example of use
```
const { setCookie,  hasCookie, deleteCookie, getCookies, getCookie,  } = useCookiesNext();
```

In my opinion it'll be better, because often we have to use several functions in one component

Reply:
What about having one hook for all of them together as you proposed and also separately as I proposed? That kind of versatility would allow developers to decide which API fits best for their code. As an example, for me, having hooks separately imported is consistent with the server-side approach.

Reply:
It makes sense